From f8a01c184ab00b048fd5413214a3d8620fe0b060 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Fri, 23 Oct 2020 12:53:35 +1000
Subject: [PATCH libinput 1/5] evdev: quirks_get_tuples can deal with a NULL
 quirks

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
(cherry picked from commit fa0c3ee38838be11a9e50cc51e4a5d42cc394b44)
---
 src/evdev.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/evdev.c b/src/evdev.c
index 40f0726b..e6a00199 100644
--- a/src/evdev.c
+++ b/src/evdev.c
@@ -2083,7 +2083,7 @@ evdev_pre_configure_model_quirks(struct evdev_device *device)
 		libevdev_disable_event_code(device->evdev, EV_MSC, MSC_TIMESTAMP);
 	}
 
-	if (q && quirks_get_tuples(q, QUIRK_ATTR_EVENT_CODE_DISABLE, &t)) {
+	if (quirks_get_tuples(q, QUIRK_ATTR_EVENT_CODE_DISABLE, &t)) {
 		int type, code;
 
 		for (size_t i = 0; i < t->ntuples; i++) {
@@ -2107,7 +2107,6 @@ evdev_pre_configure_model_quirks(struct evdev_device *device)
 	}
 
 	quirks_unref(q);
-
 }
 
 static void
-- 
2.31.1

